apiens/tools/graphql/testing/query.py

```diff
@@ -82,7 +82,11 @@ def successful(self):
         return self
 
     def __getitem__(self, name):
-        """ Get a result field, assuming that there were no errors """
+        """ Get a result field, assuming that there were no errors 
+        
+        Note: it will re-raise the original GraphQL errors if you attempt to access fields while there are errors!
+        To access fields anyway, use `.data`.
```

Comment:
Seems like this info should be on main class docstring

Reply:
Hm well, it actually should be exactly here: the `__getitem__()` function only works when you access result fields like this:

> res['fieldName']

and you only can ligitimately do this when the response has no errors.

I guess the docstring was cryptic, so I've changed it :) 
d6fd0e18b709174e8a01af84f496b622d7c896e9